protocol/shannon/mode_centralized.go

```diff
@@ -11,13 +11,16 @@ import (
 	"github.com/buildwithgrove/path/protocol/crypto"
 )
 
-// ownedApp represents a single app owned by the gateway operator in Centralized Gateway Mode.
-type OwnedApp struct {
-	// The address of the app. E.g. "pokt1..."
-	AppAddr string
-	// The service ID for which the app is staked. E.g. "anvil"
-	StakedServiceID protocol.ServiceID
-}
+// OwnedApps is a map of service IDs to a list of app addresses owned by the gateway operator in Centralized Gateway Mode.
+// One service ID can have multiple apps owned by the gateway operator.
+//
+// Example:
+//
+//	{
+//	  "anvil": ["pokt1...", "pokt2..."],
+//	  "eth": ["pokt3...", "pokt4..."],
+//	}
+type OwnedApps map[protocol.ServiceID][]string
```

Comment:
Could we avoid adding new types here: a map[protocol.ServiceID][]string is easier to read/maintain.

Reply:
Sure that's no problem; I will update.